docs/usage/configuration-options.md

```diff
@@ -377,6 +377,11 @@ Set to `null` (not recommended) to fully omit `--ignore-platform-reqs/--ignore-p
 This requires the Renovate image to be fully compatible with your Composer platform requirements in order for the Composer invocation to succeed, otherwise Renovate will fail to create the updated lock file.
 The Composer output should inform you about the reasons the update failed.
 
+## confidential
+
+If enabled, the issues and MRs created by Renovate are set as confidential, even in a public repository.
+By default issues created by Renovate are visible to all users. This option is applicable to GitLab only.
```

Comment:
One line per sentence, and do not duplicate what's already in the config options file